README.md

```diff
@@ -1,233 +1,26 @@
-`), you may keep the content of the file as it is
-1. Save this file as `winston-config.json`
-
-Now you can decide which app you want to start:
-
-* Start softwerkskammer
-
-    * `node start-softwerkskammer` - will start the server
-    * Now go to your local machine, open a browser and use `http://localhost:17124`
-
-* Start socrates
-
-    * `./build-socrates.sh` in order to build some css, js etc.
-    * `node start-socrates`- will start the server
-    * Now go to your local machine, open a browser and use `http://localhost:17224`
-
-
-Classical local installation
-=============================
-
-System requirements and Installation
---------------------
-
-### Node.js
-
-It is a node.js project. Therefore you need node.js installed. Get it from [http://nodejs.org](http://nodejs.org).
-
-You need a current 4.x version to run the software (0.12.x should also be OK).
-
-Your node.js ships a npm in suitable version.
-
-### Additional Software
-
-Some modules are compiled during the installation. Therefore some additional software must be installed.
-
-#### Python
-
-* Python 2.7 (not Python 3!) in system path
-* For Mac OS: Python 2.7 is already installed.
-* For Windows:
-  * Make sure that Python and node.js are both either 32 bit or 64 bit - mixed combinations will not work!
-  * Add an environment variable `PYTHON` that points to the Python executable.
-
-#### C++
-
-* Install a C++ compiler with headers and libraries (depending on your OS)
-  * For Mac OS X: Install XCode from the App Store. In XCode's preferences (section "Downloads"), install the command line tools. When you have installed
-    the stand alone version of the command line tools, you need to execute `sudo xcode-select -switch /usr/bin` before you are able to run `npm install`.
-  * For Windows 7 and older: Microsoft Windows SDK for Windows 7 and .NET Framework 4 [available here](http://www.microsoft.com/en-us/download/details.aspx?id=8279)
-  * For Windows 8 (may also work for Windows 7):
-     * Microsoft Visual Studio C++ 2012 for Windows Desktop ([Express](http://go.microsoft.com/?linkid=9816758) version works well)
-
-#### Git
-
-* Git must be in the system path.
-
-#### MongoDB
-
-* Install MongoDB (Version 2.4 or greater) [http://www.mongodb.org/downloads](http://www.mongodb.org/downloads)
-
-#### Grunt-CLI
+Want to help out? Check out our [Contribution guidelines](/CONTRIBUTING.md).
 
-We propose to install grunt-cli globally via the -g option of npm. To find out more about this option, see [https://npmjs.org/doc/global.html](https://npmjs.org/doc/global.html).:
+Tests
+-----
 
-* (Optional) To define the installation location of global npm packages on Unix-like Systems create a file called `.npmrc` with the following contents in your Home directory:
+Run the tests with `npm test` (after you have [set up your local installation](/INSTALL.md)).
```

Comment:
Oh, one more thing: Can we make the installation instructions a bit more prominent? (I personally would probably not bother to read a "Tests" section when looking for the installation docs, tbh)